### PR TITLE
Clean up code && Forget password task completed just API integration still remains.

### PR DESCRIPTION
**Pull request by AbdullahMuhammed5**

### CODE REVIEW

**Review by mohamedsgap — APPROVED** [review]
So far so good, @AbdullahMuhammed5  
this suggestion for all of us to consider in the next time filling PR
* if anyone make an update or built a feature in multi files, please provide some screenshots oh what already created.

**In App.js:** you remove redux boilerplate, so we gonna switch to Context API or just to keep it simple at the moment

:+1: :+1: :100:

File: src/components/CustomSnackbar.js [hunk 1]
```diff
@@ -0,0 +1,20 @@
+import React from "react";
+import Snackbar from "@material-ui/core/Snackbar";
+import Alert from "@material-ui/lab/Alert";
+
+export const CustomSnackbar = ({anchorOrigin, handleClose, open, keyProp, message}) => {
+    return (
+        <div>
```
Comment (aashrafh) [inline src/components/CustomSnackbar.js]: You don't need a ```div``` here, the ```Snackbar``` component already wraps all the children

File: src/components/Input.js [hunk 2]
```diff
@@ -0,0 +1,21 @@
+import React from "react";
+import TextField from "@material-ui/core/TextField";
+
+
+export const Input = ({variant, margin, handleChange, id, label, name, autoComplete }) => {
+    return (
+        <div>
```
Comment (aashrafh) [inline src/components/Input.js]: You don't need a ```div``` here, the ```TextField``` component already wraps all the children. Don't use divs until you really need it.

**Review by aashrafh — COMMENTED** [review]
Nice work, Abdullah. Just see the note I wrote about ```div```.

File: package.json [hunk 3]
```diff
@@ -3,6 +3,8 @@
   "version": "0.1.0",
   "private": true,
   "dependencies": {
+    "@material-ui/core": "^4.11.0",
+    "@material-ui/lab": "^4.0.0-alpha.56",
     "bootstrap": "^4.3.1",
```
Comment (ghost) [inline package.json]: `bootstrap` should be remove since you're using `material-ui` and it can affect the application loading time.

Comment (mohamedsgap) [inline package.json]:
```suggestion
```

File: src/components/CustomSnackbar.js (hunk 1 above)
Comment (mohamedsgap) [inline src/components/CustomSnackbar.js]:
```suggestion
```

File: src/components/CustomSnackbar.js [hunk 4]
```diff
@@ -0,0 +1,20 @@
...
+            anchorOrigin={ anchorOrigin } 
+            open={open} 
+            autoHideDuration={6000} 
+            onClose={handleClose}
+          >
+            <Alert onClose={handleClose} severity="success">
+              {message}
+            </Alert>
+          </Snackbar>
+        </div>
```
Comment (mohamedsgap) [inline src/components/CustomSnackbar.js]:
```suggestion
```

File: src/components/Input.js (hunk 2 above)
Comment (mohamedsgap) [inline src/components/Input.js]:
```suggestion
```

File: src/components/Input.js [hunk 5]
```diff
@@ -0,0 +1,21 @@
...
+                margin={margin}
+                fullWidth
+                onChange={handleChange}
+                id={id}
+                label={label}
+                name={name}
+                autoComplete={autoComplete}
+                autoFocus
+            />
+        </div>
```
Comment (mohamedsgap) [inline src/components/Input.js]:
```suggestion
```

**Review by mohamedsgap — CHANGES_REQUESTED** [review]
@AbdullahMuhammed5 after our fellows suggestions, you can commit my changes as they asked directly, you don't need to do it manually on VS Code

**Review by mohamedsgap — DISMISSED** [review]
LGTM

**Review by aashrafh — DISMISSED** [review]

**Review by mohamedsgap — DISMISSED** [review]
Made a commit to fix some issues, after @AbdullahMuhammed5  commit the suggestions of other fellows 
:+1: :100:

**Review by ghost — DISMISSED** [review]

**Review by aashrafh — DISMISSED** [review]

**Comment by mohamedsgap** [comment]
@AbdullahMuhammed5 I can see that you removed `Material UI` deps why ? and there is still an issue, so `Netlify` didn't pass!!

**Review by mohamedsgap — APPROVED** [review]
Resolved all conflicts :) :+1:

**Review by ghost — APPROVED** [review]

**Review by RanaEmad — APPROVED** [review]

**Pull request merged**